gdk4-win32/src/win32_display.rs

```diff
@@ -0,0 +1,26 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::Win32Display;
+#[cfg(any(feature = "wayland_crate", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "wayland_crate")))]
+use glib::translate::ToGlibPtr;
+#[cfg(any(all(feature = "v4_4", feature = "egl"), feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(all(feature = "v4_4", feature = "egl"))))]
+use khronos_egl as egl;
+
+impl Win32Display {
+    #[cfg(any(all(feature = "v4_4", feature = "egl"), feature = "dox"))]
```

Comment:
the gir files doesn't seem to annotate this as being available only in v4_4?

Reply:
it's not annotated correctly,  now part of https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4980